specification/gedcom.md

```diff
@@ -316,6 +316,24 @@ Each non-pointer payload may be encoded in 1 or more line strings (line continua
 The exact encoding of non-pointer payloads is dependent on the datatype of the payload, as determined by the structure type.
 The datatype of non-pointer payloads cannot be fully determined by line value content alone.
 
+Note that production `LineVal` does not match the empty string.
+Because empty payloads and missing payloads are considered equivalent,
+both a structure with no payload
+and a structure with the empty string as its payload
+are encoded with no `LineVal` and no space after the `Tag`.
+
+:::example
+The payload of a `MARR` structure has type `[Y|<NULL>]`, which is optional but cannot be the empty string.
+The payload of a `EVEN` structure has type `Text`, which is not optional but can be the empty string.
+The `Line` encoding a no-payload `MARR` is "`1 MARR`"
+and the `Line` encoding an empty-payload `EVEN` is "`1 EVEN`";
+both `Line`s have no `LineVal` and no trailing space.
+:::
+
+:::note
+Implementations may choose to be more permissive and parse lines ending with a `Tag` and a single space as if they had no `LineVal`, but are not required to do so. They must not create lines of this form.
+:::
```

Comment:
I'm ok with this note, but I don't think it's necessary and would prefer removing it.

Reply:
Rationale for removal is that there's lots of other ways throughout the spec that implementations may choose to be more permissive and parse lines that should never occur, but we don't have notes for them.

Reply:
```suggestion
```